www/shared/test/utils/render-with-intl/render-with-intl.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { renderWithIntl } from './render-with-intl';
+
+describe('Render with Intl', () => {
+    it('should correctly render formatted message', () => {
+        const { container, getByText } = renderWithIntl(
+            <FormattedMessage id={ 'foo' } />,
+        );
+
+        expect(container.innerHTML).toMatchSnapshot();
+        expect(getByText('foo')).toBeInTheDocument();
+    });
+
+    it('should correctly rerender', () => {
+        const render = renderWithIntl(
+            <FormattedMessage id={ 'foo' } />,
+        );
+
+        render.rerenderWithIntl('bar');
+
+        expect(render.container.innerHTML).toMatchSnapshot();
```

Comment:
Could you please use `getByText` here as well? I personally think we should avoid snapshots if the content is small, which it is in this case. 

Also, i would suggest destructuring `container`, `renderWithIntl` et al, like the test above. I think it looks prettier and easier to read!

Reply:
Will do! Thanks!

Reply:
Will do! Thanks.